app/__init__.py

```diff
@@ -100,15 +104,21 @@ async def notify(
     return data
 
 
-async def get_user_list(db_session: AsyncSession) -> list[Registration]:
-    query = select(Registration)
+async def get_registration_list(db_session: AsyncSession) -> list[Registration]:
+    query = select(Registration).order_by(desc(Registration.email))
+    result = await db_session.exec(query)
+    return list(result.all())
+
+
+async def get_notification_list(db_session: AsyncSession) -> list[Notification]:
+    query = select(Notification).order_by(desc(Notification.date))
```

Comment:
Sadly, this gives a type error:

But it can be fixed with the following change:
```suggestion
    query = select(Notification).order_by(col(Notification.date).desc())
```

It's a bit frustrating that we have several ways to do the same thing, and use different solutions throughout the code, not very pythonic :/
Eg:
- `query = select(Notification).order_by(desc(Notification.date))`
- `query = select(Notification).order_by(Notification.date.desc())`
- `query = select(Notification).order_by(col(Notification.date).desc())`

Maybe we should be using `col()` everywhere for consistency, as we're already using it for:
`query = select(Notification).where(col(Notification.email) == email)`
(actually, at least according to pyright, entirely removing the `col()` from this line is fine).

Reply:
I agree that it's not ideal, adding `col()` everywhere for consistency adds an overhead but I can live with it because it's also more explicit than the magic SQLAlchemy is doing here. I'll uniformise that 👍